client/config/webpack.config.js

```diff
@@ -152,8 +162,8 @@ function config(webpackEnv) {
       level: "none",
     },
     optimization: {
-      chunkIds: isEnvProduction ? "natural" : "named",
-      moduleIds: isEnvProduction ? "natural" : "named",
+      chunkIds: isEnvProduction ? "deterministic" : "named",
+      moduleIds: isEnvProduction ? "deterministic" : "named",
```

Comment:
Out of curiosity: What difference does it make?

Reply:
"deterministic" is the default behaviour where if one commit you have code like:
```
import("x.js")
```
and the next:
```
import("y.js")
import("x.js")
```
`x.js`'s chunk will still have the same name (like `33.chunk.js`) or something.

With "natural", chunks are named based on their ordering, so in the previous example originally `x.js`'s chunk is named `1.chunk.js`, but then `y.js`'s chunk becomes `1.chunk.js` and `x.js`'s chunk becomes `2.chunk.js`. This makes comparing the size diffs horrible, because it seems like the same chunk varies massively in size between PRs, when actually it's just a different chunk.

I originally made the change to "natural" because the one downside of "deterministic" is if you change the loader module order (or possibly even config) - as I was when cleaning up some webpack things - the deterministic id changes, even if the actual contents of the chunk doesn't, which made diffing before/after the webpack config change very difficult. Next time I need to do that, I'll only change to "natural" locally when diffing.

Reply:
Thanks for the clarification!